### PR TITLE
Add example for Mocking resources in v2.2

### PR DESCRIPTION
**Pull request by fermujica**
This example was missing from the v2.2 cookbook

### CODE REVIEW

File: modules/ROOT/pages/mock-file-cookbook.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+= Mocking Resources for Your Tests
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As your tests suites start to get bigger and more complex, it’s a good practice to externalize some parts of the test code, not only so that you can keep the code clean and organized but also so that you can reuse the files that you need to mock for your tests.
```
Comment (diegostrubolini) [inline modules/ROOT/pages/mock-file-cookbook.adoc]: I feel too many "so that you can" what do you think @fermujica

**Review by diegostrubolini — DISMISSED** [review]

File: modules/ROOT/pages/mock-file-cookbook.adoc (hunk 1 above)
Comment (fermujica) [reply modules/ROOT/pages/mock-file-cookbook.adoc]: Agreed.

File: modules/ROOT/pages/mock-file-cookbook.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+</munit-tools:then-return>
+----
+
+The `classpath-based` URL uses the `classpath:` protocol prefix to find the file defined in `src/test/resources`.
+
+== Mocking Files Using Variables in a DataWeave Module
+
+You can create a DataWeave module to centralize all your mocked data in a single file by declaring it using DataWeave variables.
+
+For example, imagine that you create a custom DataWeave module by creating a `TestData.dwl` file in `src/test/resources`, and you add a couple of variables with references to different mocked data, either from files or inline:
```
Comment (dhirsch1) [inline modules/ROOT/pages/mock-file-cookbook.adoc]: Usually "a couple of" means 2. I would remove it and just say "add variables with..."

**Review by dhirsch1 — APPROVED** [review]
Approved with one comment

**Pull request merged**